cypress/integration/Stepper._spec.js

```diff
@@ -0,0 +1,227 @@
+/* This Source Code Form is subject to the terms of the Creative Commons
+ * License Chooser
+*/
+
+// test the interpolated library
+
+describe('Stpper.vue', () => {
+    it('renders correctly', () => {
+        cy.visit('/')
+    })
+
+    describe('Checks the user know the licence needed', () => {
+        it('Checks whether the CC0 license is selected in the dropdown', () => {
+        cy.get('.step-0')
+            .find('[value="yes"]')
+            .check()
+        cy.get('.next-button').click()
+        cy.get('select').select('CC0 1.0')
+        cy.get('.recommended-card').should('be.visible').contains('CC0 1.0 Universal')
+        cy.get('.next-button').click()
+        cy.get('.v-checkbox:nth-child(1) > input').check()
```

Comment:
Can you add a check for `and the total number of steps should be 4. `

Reply:
@obulat  I have a question that whether total number of steps are required for the test which you have mentioned or for each and every test in the test file.

Reply:
It's for this test only.